core/src/test_config.rs

```diff
@@ -0,0 +1,3 @@
+pub const ISSUER_URL: &str = "https://mongodb-dev.okta.com/oauth2/ausqrxbcr53xakaRR357";
```

Comment:
Is this your own test instance?

Reply:
It's *our* test instance

Reply:
If you actually want to check powerbi you'd use mongodb://sqlenginesoidctest-lxonw.a.query.mongodb-dev.net/?directConnection=true&tls=true&authMechanism=MONGODB-OIDC&appName=mongosh+2.2.5

That is only for testing the callbacks

Reply:
oh, I just wanted to make sure that this is a shared instance we have to maintain.
We should document things in our Wiki. Who is admin, how to access it not from PowerBI to make updates if needed, etc...

Reply:
That is an okta account set up for what appears to be all of mongodb employees, or at least engineers.